src/annotator/integrations/html.ts

```diff
@@ -22,6 +22,11 @@ import {
 // content line lengths and scale are too short to be readable.
 const MIN_HTML_WIDTH = 480;
 
+type SideBySideOptions = {
+  minWidth: number;
+  mode: 'auto' | 'all' | 'off'; // TODO Review possible values
```

Comment:
For the video player app, I think the plan is that the video player will completely take over applying side-by-side mode. For that need we can simplify the options to something like `{ mode: 'auto' | 'manual' }`. I used "manual" here because side-by-side will still be used, but it will be up to the host page to apply it "manually" by responding to `hypothesis:layoutchange` events.

I don't think the `minWidth` option is needed for the video player any more. It might turn out to be useful for other apps in future, if they want to use the `auto` mode but customize just that aspect of the automatic behavior. In that case the type would be something like:

```ts
type SideBySideOptions = { mode: 'auto', minWidth?: number } | { mode: 'manual' }
```